package.json

```diff
@@ -68,8 +66,10 @@
     "ts-node": "^8.10.2",
     "tslint-config-prettier": "^1.18.0",
     "tslint-config-standard": "^8.0.1",
-    "typescript": "^4.4.2",
-    "validate-commit-msg": "^2.14.0"
+    "typescript": "^4.4.2"
   },
-  "dependencies": {}
+  "dependencies": {},
+  "resolutions": {
+    "trim": "0.0.3"
```

Comment:
How does this work for consumers who use e.g. npm? Would adding the `trim` as a direct dependency instead help?

Reply:
Direct dependency sadly doesn't change much, as the version is pinned to 0.0.1 by remark-parse.
With the resolution, we get a warning, but at least it will be patched.
I tested it, by importing the package once locally in another repo, patched the resolutions and imported it again, and it removed the security warning in the other repo.
For this repo I needed to define the resolution twice, once for the package (will be applied by installing the package) and once for the root (fixes the warning in this repo).
Will also have a quick look, how much effort it would be to upgrade to remark-parse with the correct trim version

Reply:
Ok. Got it. Maybe we can also add [npm `overrides`](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides) in addition to yarn resolutions?